webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7063,7 +7063,7 @@ var userOverride = {
                 isAgentOffline
             ) {
                 var userSession = KommunicateUtils.isSessionStorageAvailable()
-                    ? JSON.parse(sessionStorage.kommunicate)
+                    ? (sessionStorage.kommunicate ? JSON.parse(sessionStorage.kommunicate) : {})
```

Comment:
`KommunicateUtils.isSessionStorageAvailable()` vs `sessionStorage.kommunicate`
whats the difference?

Reply:
sessionStorage is available but it doesn't have `kommunicate` in storage
thus `kommunicate` becomes undefined and `JSON.parse(undefined)` gives error

Reply:
any ideas why its not present?

Reply:
The customers' website was using some tool to ask user permissions related to cookie and storage
so when the user visits for the 1st time there was no data in storage and also no permission to write data.